gateway/api/models.py

```diff
@@ -218,3 +218,4 @@ class RuntimeJob(models.Model):
     runtime_job = models.CharField(
         primary_key=True, max_length=100, blank=False, null=False
     )
+    session_id = models.CharField(max_length=100, blank=True, null=True, default=None)
```

Comment:
I would like to maintain consistency in the names here `runtime_job`, `runtime_session` (I would agree that probably would better to have `job_id`, `session_id` but I don't think we would need to do a migration due to this).

Reply:
✅